gammapy/datasets/map.py

```diff
@@ -475,18 +476,21 @@ def _background_parameters_changed(self):
             self._background_parameters_cached = values
         return changed
 
-    def npred_signal(self, model_name=None):
+    @deprecated_renamed_argument("model_name", "model_names", "1.1")
+    def npred_signal(self, model_names=None, stack=True):
```

Comment:
Do we need a `stack` option here? Users can do `dataset.npred_signal(model_names).stack()` if necessary. This is the general scheme we follow, no?

Reply:
I have done this to avoid to have to rewrite the code everywhere where `npred_signal()` is called. 
That's why it is set to `True` by default, so that by default it behave exactly as before.

Reply:
Yes indeed, I forgot this is necessary for backwards compatibility.

Reply:
Having `stack=True` by default also save memory.